### PR TITLE
[All Hosts] (manifest) supplementary reference for requirements property

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
This was reviewed when it was in a different PR.

This article is deliberately left out of the ToC because it will be moved to the M365 doc set in a few weeks.

### CODE REVIEW

File: docs/develop/requirements-property-unified-manifest.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+- Either hide or give preference to the version that uses the older technology.
+- Specify icons that are needed to make the unified manifest version of the add-in installable on Office versions that don't directly support the unified manifest. 
+
+> [!NOTE]
+> Office Add-ins that use the unified manifest for Microsoft 365 are *directly* supported in Office on the web, in [new Outlook on Windows (preview)](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627), and in Office on Windows connected to a Microsoft 365 subscription, Version 2304 (Build 16320.00000) or later.
+>
+> When the app package that contains the unified manifest is deployed in [AppSource](https://appsource.microsoft.com/) or the [Microsoft 365 Admin Center](../publish/publish.md) then, if the manifest has a valid "alternateIcons" property, an XML manifest is generated from the unified manifest and stored. This XML manifest enables the add-in to be installed on platforms that don't directly support the unified manifest, including Office on Mac, Office on mobile, subscription versions of Office on Windows earlier than 2304 (Build 16320.00000), and perpetual versions of Office on Windows.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/requirements-property-unified-manifest.md:64]:
```suggestion
> When the app package that contains the unified manifest is deployed in [AppSource](https://appsource.microsoft.com/) or the [Microsoft 365 Admin Center](/office/dev/add-ins/publish/publish) then, if the manifest has a valid "alternateIcons" property, an XML manifest is generated from the unified manifest and stored. This XML manifest enables the add-in to be installed on platforms that don't directly support the unified manifest, including Office on Mac, Office on mobile, subscription versions of Office on Windows earlier than 2304 (Build 16320.00000), and perpetual versions of Office on Windows. 
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

File: docs/develop/requirements-property-unified-manifest.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                }
+            }
+        ]
+    }
+]
+```
+
+### extensions.runtimes.requirements
+
+The "extensions.runtimes" property configures the sets of runtimes and actions that each extension point can use. For more information on its usage, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command). For more information about runtimes in Office add-ins, see [Runtimes in Office Add-ins](/office/dev/add-ins/testing/runtimes).
```
Comment (lindalu-MSFT) [inline docs/develop/requirements-property-unified-manifest.md:169]:
```suggestion
The "extensions.runtimes" property configures the sets of runtimes and actions that each extension point can use. For more information on its usage, see [Create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [Configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [Configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command). For more information about runtimes in Office add-ins, see [Runtimes in Office Add-ins](/office/dev/add-ins/testing/runtimes).
```

**Review by lindalu-MSFT — APPROVED** [review]
Some minor suggestions, otherwise looks good!

File: docs/develop/requirements-property-unified-manifest.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+```json
+"extensions": [
+    ...
+    {
+        ...
+        "ribbons": [
+            ...
+            {
+                // Details of the ribbon configuration would be here.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/develop/requirements-property-unified-manifest.md:151]:
```suggestion
                // Insert details of the ribbon configuration here.
```

File: docs/develop/requirements-property-unified-manifest.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+```json
+"extensions": [
+    ...
+    {
+        ...
+        "runtimes": [
+            ...
+            {
+                // Details of the runtime configuration would be here.
```
Comment (Rick-Kirkham) [inline docs/develop/requirements-property-unified-manifest.md:184]:
```suggestion
                // Insert details of the runtime configuration here.
```

**Pull request merged**